mm2src/coins/utxo/utxo_withdraw.rs

```diff
@@ -104,8 +104,10 @@ where
 {
     fn coin(&self) -> &Coin;
 
+    #[allow(clippy::wrong_self_convention)]
     fn from_address(&self) -> Address;
 
+    #[allow(clippy::wrong_self_convention)]
     fn from_address_string(&self) -> String;
```

Comment:
We can rename if required. (New warnings from cargo after updating to the last toolchain)

Reply:
Please solve such Clippy warnings instead of allowing them.